routes/api/v1/foods.js

```diff
@@ -23,9 +23,23 @@ router.get('/:id', (req, res) => {
       res.status(404).send(JSON.stringify({error: "Food with ID(" + req.params.id + ") not found."}));
     }
   })
-  .catch(error => {
-    res.status(500).send(JSON.stringify(error));
+  .catch(error => res.status(500).send(JSON.stringify(error)));
+});
+
+router.patch('/:id', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+  Food.update(
+    {name: req.body.name,
+      calories: req.body.calories
+    }, {where: {id: req.params.id}, returning: true})
+  .then(([row, [food]]) => {
```

Comment:
Later, can you explain this line of code to me? I'm just interested in what's it doing and haven't seen this before.